requirements.txt

```diff
@@ -0,0 +1,7 @@
+asgiref==3.7.2
+Django==4.2.4
+django-appconf==1.0.5
+python-dotenv==1.0.0
+six==1.16.0
+sqlparse==0.4.4
+typing_extensions==4.7.1
```

Comment:
dalej nie poprawione, tutaj dajemy tylko te paczki które normalnie byśmy instalowali przez pip install, a taki six np się instaluje bo inne paczki go wymagają, ale nie jest to pomocne w czytaniu requirements potem